src/utils/domain.ts

```diff
@@ -22,11 +30,13 @@ export const APP_URL =
 export const STORAGE_URL = (process.env
   .REACT_APP_S3_BASE_URL as string).replace(/\/$/, '');
 export const API_URL = process.env.REACT_APP_API_BASE_URL as string;
+export const API_IS_PRODUCTION = API_URL === 'https://coordinape.me/api';
```

Comment:
this should not be hardcoded. you can use process.env.NODE_ENV for this ([docs](https://create-react-app.dev/docs/adding-custom-environment-variables/))